llvm/include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -585,6 +586,11 @@ class AsmPrinter : public MachineFunctionPass {
     emitGlobalConstant(DL, CV);
   }
 
+  /// Lower the specified ptrauth constant to an MCExpr.
```

Comment:
nit: This comment doesn't add anything over the function prototype

Reply:
Makes sense, deleted the comment, see 9633e04f21f0745dc76f905684168333c2c4ff4d